tests/core/test_curate.py

```diff
@@ -215,13 +243,49 @@ def test_clean_up_failed_runs():
 
 @pytest.mark.parametrize("to_add", ["donor", "all"])
 def test_anndata_curator(adata, categoricals, to_add):
+    adata = adata.copy()
+    # must pass an organism
+    with pytest.raises(ValidationError):
+        bt.settings._organism = None  # make sure organism is not set globally
```

Comment:
This involves a lot of changes, will do in a different PR.

Reply:
Why? Isn't it something like:

```
old_organism = bt.settings._organism
bt.settings._organism = None
# more code
bt.settings._organism = old_organism
```

?

Reply:
Because there's this one: https://github.com/laminlabs/lamindb/blob/2e298635f61ee745acf41fb66b3d711eb79c1726/tests/core/test_artifact.py#L45

I need to go through all the test functions to see which ones needs it.